openreview/venue/process/post_submission_process.py

```diff
@@ -29,7 +29,7 @@ def post_submission_edit(submission):
                         year=str(datetime.datetime.utcnow().year),
                         url_forum=submission.forum,
                         paper_status='under review',
-                        anonymous='readers' in submission.content['authors']
+                        anonymous='readers' in submission.content['authors'] or 'readers' in invitation.edit.get('note', {}).get('content', {}).get('authors', {})
```

Comment:
I don't understand which issue is fixing this.

Reply:
The problem happened in a venue that wanted to set the readers of the Note to 'everyone', but hide the authors and authorids field by setting a readers field.

The problem is that because the authors field when posting the change didn't have a readers field in the Submission, the process function "thinks" that the note is non-anonymous and adds the authors' names to the bibtex even if the post edit hides them by adding a readers field. So the fix is to check if the readers is a constant in the invitation that will hide authors to avoid revealing them in the bibtex.

Reply:
So this happens when you want to release the submission to the public and hide the authors identities at the same time using the same edit. 

Could you add a test to reproduce this issue? I think this is a common case for ICLR venues. I can help writing the test if you need.

Reply:
Yes